addons/website/models/website_page.py

```diff
@@ -43,6 +43,33 @@ class Page(models.Model):
     website_id = fields.Many2one(related='view_id.website_id', store=True, readonly=False, ondelete='cascade')
     arch = fields.Text(related='view_id.arch', readonly=False, depends_context=('website_id',))
 
+    # Bindings to model/records, to expose the page on the website.
+    # Route: /model/<string:page_name_slugified
+    page_name = fields.Char(string="Page Name", help="The name of the page")
+    name_slugified = fields.Char(compute="_compute_page_url", store=True,
```

Comment:
I am not sure about this but I think this will change as right now there is pages with dynamic/converter in their route, which breaks the page manager and maybe the sitemap and url autocomplete too (see screenshot, those urls are returned)
![image](https://github.com/odoo/odoo/assets/30048408/58d5ad47-369b-48b2-970f-1744f8b813d3)